README.md

```diff
@@ -138,7 +140,7 @@ def index():
         ...
     )
 ```
-This index function defines the frontend of the app. We use different components such as center, vstack, input, and button to build the front end. Components can be nested to create complex layouts and styled using CSS's full power. Just pass them in as keyword args.
+This index function defines the frontend of the app. We use different components such as ```center```, ```vstack```, ```input```, and ```button``` to build the front end. Components can be nested to create complex layouts and styled using CSS's full power. Just pass them in as keyword args.
```

Comment:
Thanks for the updates! (I think you can use just one backtick in the future for inline, and 3 for a big code block)